openoffice.txt

```diff
@@ -1,102 +1,89 @@
-SizeBytes = 136182684
+URLSite = https://www.openoffice.org/it/
+URLDownload = https://download.sourceforge.net/project/openofficeorg.mirror/4.1.10/binaries/it/Apache_OpenOffice_4.1.10_Win_x86_install_it.exe
+SHA1 = e29315985b8a230e22a6b1b634935c73664485d3
+SizeBytes = 137653816
 
 [Section.0413]
 Description = Open-bron Office Pakket.
-Size = 135.6 MiB
-URLSite = http://www.openoffice.org/nl/
-URLDownload = http://download.sourceforge.net/project/openofficeorg.mirror/4.1.5/binaries/nl/Apache_OpenOffice_4.1.5_Win_x86_install_nl.exe
-SHA1 = 686fef509d238683818d160b9972d29ef31ce9d9
-SizeBytes = 142170939
+URLSite = https://www.openoffice.org/nl/
+URLDownload = https://download.sourceforge.net/project/openofficeorg.mirror/4.1.10/binaries/nl/Apache_OpenOffice_4.1.10_Win_x86_install_nl.exe
+SHA1 = 2822e94418ad794a4107f722c649319fff04dd55
+SizeBytes = 143513280
 
 [Section.0415]
 Description = Otwarty pakiet biurowy.
-Size = 125.4 MiB
-URLSite = http://www.openoffice.org/pl/
-URLDownload = http://download.sourceforge.net/project/openofficeorg.mirror/4.1.5/binaries/pl/Apache_OpenOffice_4.1.5_Win_x86_install_pl.exe
-SHA1 = 7011d7621ee53ea43537d25c5f676806975238e4
-SizeBytes = 131496279
+URLSite = https://www.openoffice.org/pl/
+URLDownload = https://download.sourceforge.net/project/openofficeorg.mirror/4.1.10/binaries/pl/Apache_OpenOffice_4.1.10_Win_x86_install_pl.exe
+SHA1 = 6780c4f63df808f8081c590dafa66af6a798c753
+SizeBytes = 132890808
 
 [Section.0418]
 Description = Suită de aplicații de birotică cu surse deschise.
-URLSite = http://www.openoffice.org/ro/
-Size = 134,3 Mio
+URLSite = https://www.openoffice.org/ro/
 
 [Section.0419]
 Description = Офисный пакет с открытым исходным кодом.
-URLSite = http://www.openoffice.org/ru/
-URLDownload = http://download.sourceforge.net/project/openofficeorg.mirror/4.1.5/binaries/ru/Apache_OpenOffice_4.1.5_Win_x86_install_ru.exe
-Size = 133,5 МиБ
-SHA1 = e8678df898ee58e6f7b14e7298a28bc4e7599236
-SizeBytes = 139972276
+URLSite = https://www.openoffice.org/ru/
+URLDownload = https://download.sourceforge.net/project/openofficeorg.mirror/4.1.10/binaries/ru/Apache_OpenOffice_4.1.10_Win_x86_install_ru.exe
+SHA1 = 56eca7cc4aa90aefb560bc490c04a04e03459894
+SizeBytes = 141311112
 
 [Section.041f]
 Description = Bir açık kaynak ofis paketi.
-Size = 122,3 MiB
-URLSite = http://www.openoffice.org/tr/
-URLDownload = http://download.sourceforge.net/project/openofficeorg.mirror/4.1.5/binaries/tr/Apache_OpenOffice_4.1.5_Win_x86_install_tr.exe
-SHA1 = 376c5e0664e6d7db613db70fa1b06a2bcbd4ebf1
-SizeBytes = 128251698
+URLSite = https://www.openoffice.org/tr/
+URLDownload = https://download.sourceforge.net/project/openofficeorg.mirror/4.1.10/binaries/tr/Apache_OpenOffice_4.1.10_Win_x86_install_tr.exe
+SHA1 = f4555bcb48018032a151dd8ef068362b97dc341e
+SizeBytes = 129619608
 
 [Section.0422]
 Description = Відкритий офісний пакет.
-URLSite = http://www.openoffice.org/ua/
+URLSite = https://www.openoffice.org/uk/
```

Comment:
`ua` is not `uk`, is this an intentional change?

Reply:
Yes, uk is the language code for Ukrainian. ua is the country code for the Ukraine.
OpenOffice only supports languages, not countries.
That was fixed on the homepage.

Reply:
Confirming, `/ua/` leads to 404 page.